src/object.c

```diff
@@ -1263,6 +1263,11 @@ struct redisMemOverhead *getMemoryOverheadData(void) {
         mem_total+=mem;
 
         mh->num_dbs++;
+
+        mh->db_overhead_hashtable_lut += kvstoreOverheadHashtableLut(db->keys);
+        mh->db_overhead_hashtable_lut += kvstoreOverheadHashtableLut(db->expires);
```

Comment:
It's better to also calculate the `overhead_hashtable_rehashing` and display it in the stats results.

Reply:
So `overhead_hashtable_rehashing` in both `info memory` and `memory stats`, and the other two only in `memory stats`? Perhaps we need to convince @madolson about this ...